functions/Copy-DbaDatabase.ps1

```diff
@@ -1061,15 +1061,15 @@ function Copy-DbaDatabase {
 						$backupResult = $BackupTmpResult.BackupComplete
 						if ($backupResult -eq $false) {
 							$serviceAccount = $sourceServer.ServiceAccount
-							Write-Message -Level Verbose -Message "Backup Failed. Does SQL Server account $serviceAccount have access to $NetworkShare ? Aborting routine for this database."
+							Write-Message -Level Verbose -Message "Backup Failed. Does SQL Server account $serviceAccount have access to '${NetworkShare}'? Aborting routine for this database."
 							
 							$copyDatabaseStatus.Status = "Failed"
-							$copyDatabaseStatus.Notes = "Backup failed. Verify service account access to $NetworkShare"
+							$copyDatabaseStatus.Notes = "Backup failed. Verify service account access to '${NetworkShare}'."
```

Comment:
In these cases we have been using `$($variable)` and not curly braces. If you can use that same pattern it would help keep to the standard.

Reply:
Perfect, no problem.